container_files/assets/www/doc/builtin/functions/Tfidf.md

```diff
@@ -7,8 +7,39 @@ appears in a reference corpus.
 ## Configuration
 ![](%%config function tfidf)
 
+### Type of TF scoring
+
+Given a term \\( t \\), a document \\( d \in D \\) and the term frequency denoted by \\(f_{t,d}\\):
```

Comment:
You might want to introduce the table.  Something like _the type of TF scoring are:_